### ISSUE TITLE
Links always underlined

### ISSUE THREAD

**Issue by cpytel** [thoughtbot/constable#646]
The links in the interests navigation and announcement titles are always underlined, and it seems like this is incorrect.

<img width="858" alt="Screen Shot 2019-08-03 at 2 30 12 PM" src="https://user-images.githubusercontent.com/5015/62415663-82f4a280-b5fb-11e9-9005-369e93d2d0f4.png">

**Comment by entcheva** [comment]
This change is intentional, introduced in https://github.com/thoughtbot/constable/pull/639.

Previously there was no visual distinction of links for those with certain types of color blindness. I'm certainly open to improving the approach!

**Comment by cpytel** [comment]
Sorry, I’m not sure I understand how the items I put the arrows on relate to color blindness.

For the item in the the navigation there is a black visual distinction that this is the page you are currently on. I wouldn’t expect this link to be underlined as well, as that is duplicate info and clicking that link takes you to the place you were already on.

In general, when a link has no underlines and underlines appear on hover, as was the previous behavior throughout constable, I see how one could say that isn’t intuitive, but I’m not sure how it relates to color blindness.

For the announcement title links, they were previously just all black with no underline, and then an underline on hover. Isn’t this the same distinction for both color blind and non-color blind folks?

Very happy to learn more about this. Thanks.

**Comment by entcheva** [comment]
> In general, when a link has no underlines and underlines appear on hover, as was the previous behavior throughout constable, I see how one could say that isn’t intuitive, but I’m not sure how it relates to color blindness.

@cpytel Protanopia is a type of color blindness where people affected have limited perception of the color red. Previously, Constable body text links were red with no underline, and for people with protanopia, it was not clear where to hover because that link's color difference is not perceived. This came up in a Constable thread here: https://constable.io/announcements/2101-nyc-quarterly-outing-summer-2019-sept-13th#comment-11339

Re: navigation items - Yes, I see that there's a black visual distinction in the navigation already, so I agree that the underline is not needed there.

Re: announcement title links - I like your suggestion here to make the underline the same color as the text: https://github.com/thoughtbot/constable/issues/653.
